src/http.rs

```diff
@@ -67,15 +73,31 @@ pub async fn routes() -> impl Filter<Extract = impl warp::Reply, Error = warp::R
     let all_docs_route = warp::path!(String / "_all_docs")
         .and(warp::get())
         .and(with_fdb(fdb.clone()))
-        .and(with_couch_directory(couch_directory))
+        .and(with_couch_directory(couch_directory.clone()))
         .and_then(all_docs_req);
 
+    let changes_route = warp::path!(String / "_changes")
+        .and(warp::get())
+        .and(with_fdb(fdb.clone()))
+        .and(with_couch_directory(couch_directory.clone()))
+        .and_then(changes_req);
```

Comment:
You might have to add a `warp::path::end()` at the end of this so that the path below works.